packages/next/src/server/lib/router-utils/setup-dev-bundler.ts

```diff
@@ -664,6 +665,10 @@ async function startWatcher(
             opts.fsChecker.rewrites.beforeFiles.length > 0 ||
             opts.fsChecker.rewrites.fallback.length > 0
 
+          const rootPath =
+            opts.nextConfig.turbopack?.root ||
```

Comment:
nit: `??` instead of `||` 

empty string is probably an error, right? so we shouldn't fall back

Reply:
yeah but it is an obvious user error, right? so breakage is good?  

this is preexisting behavior i guess so not really a big deal, just didn't seem intentional